### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: bin/tSmoke [hunk 1]
```diff
@@ -282,7 +282,7 @@ sub Summary_Sheet() {
   $Body .= "<td class ='appHeader' colspan='5'>Compiled: ". scalar(localtime) . "</td></tr>\n";
   $Body .= "<tr>\n";
   $Body .= "<td class = 'subhead' width='20%'>Service</td>
-            <td class = 'subhead' witdh='20%'>Past Quarter</td>
+            <td class = 'subhead' width='20%'>Past Quarter</td>
```
Comment (jsoref) [inline bin/tSmoke]: This may constitute a bug fix / change output

File: etc/smokeping_secrets.dist [hunk 2]
```diff
@@ -1,3 +1,3 @@
-host1:mysercert
-host2:yoursercert
+host1:mysecret
```
Comment (jsoref) [inline etc/smokeping_secrets.dist]: I'm assuming this isn't a problem

File: lib/Smokeping.pm [hunk 3]
```diff
@@ -35,7 +35,7 @@ $ENV{'LC_NUMERIC'}='C';
 if (setlocale(LC_NUMERIC,"") ne "C") {
     if ($ENV{'LC_ALL'} eq 'C') {
         # This has got to be a bug in perl/mod_perl, apache or libc
-        die("Your internalization implementation on your operating system is "
+        die("Your internationalization implementation on your operating system is "
```
Comment (jsoref) [inline lib/Smokeping.pm]: I believe this is correct

File: lib/Smokeping.pm [hunk 4]
```diff
@@ -3310,12 +3310,12 @@ DOC
            hierarchies => {
               _doc => <<DOC,
 Provide an alternative presentation hierarchy for your smokeping data. After setting up a hierarchy in this
-section. You can use it in each tagets parent property. A drop-down menu in the smokeping website lets
+section. You can use it in each target's parent property. A drop-down menu in the smokeping website lets
```
Comment (jsoref) [inline lib/Smokeping.pm]: I hope that an `'` isn't a problem, I'm pretty sure this is a correct fix otherwise

File: lib/Smokeping.pm [hunk 5]
```diff
@@ -3950,7 +3950,7 @@ Its location must be hardcoded in the smokeping script and smokeping.cgi.
 The contents of this manual is generated directly from the configuration
 file parser.
 
-The Parser for the Configuration file is written using David Schweikers
+The Parser for the Configuration file is written using David Schweikert
```
Comment (jsoref) [inline lib/Smokeping.pm]: This matches the other instance of his name in this project and the project per CPAN

File: lib/Smokeping/RRDtools.pm [hunk 6]
```diff
@@ -209,7 +209,7 @@ sub tuneds {
 	       next unless $ds[1] eq $ds2[1] and $ds[2] eq $ds[2];
 	       if ($ds[3] ne $ds2[3]){
 	       		warn "## Updating $file DS:$ds[1] heartbeat $ds2[3] -> $ds[3]\n";
-	  	        RRDs::tune $file,"--hearbeat","$ds[1]:$ds[3]" unless $ds[3] eq $ds2[3];
+	  	        RRDs::tune $file,"--heartbeat","$ds[1]:$ds[3]" unless $ds[3] eq $ds2[3];
```
Comment (jsoref) [inline lib/Smokeping/RRDtools.pm]: This could either be a bug fix or break something....

File: lib/Smokeping/matchers/ConsecutiveLoss.pm [hunk 7]
```diff
@@ -123,11 +123,11 @@ sub Test($$) {
     # Determine number of iterations for the for-loop. if we at all have enough values yet.
     if ( $data->{prevmatch} ) {
         # Alert state true
-        if (scalar @{ $data->{loss} } < $self->{param}{stepsclear})  { return $data->{prevmatch}; } # Cannot consider $stepclear values unless so many values actuelly exist in array
+        if (scalar @{ $data->{loss} } < $self->{param}{stepsclear})  { return $data->{prevmatch}; } # Cannot consider $stepsclear values unless so many values actually exist in array
```
Comment (jsoref) [inline lib/Smokeping/matchers/ConsecutiveLoss.pm]: These two changes are to match the actual variables in this code

File: lib/Smokeping/probes/SendEmail.pm [hunk 8]
```diff
@@ -81,16 +81,16 @@ sub probevars {
 		},
 		to => {
 			_doc => "Rcpt to address",
-			_exemple => 'test@test.com',
+			_example => 'test@test.com',
```
Comment (jsoref) [inline lib/Smokeping/probes/SendEmail.pm]: I believe this is a bug fix

File: lib/Smokeping/sorters/Loss.pm [hunk 9]
```diff
@@ -18,7 +18,7 @@ Call the sorter in the charts section of the config file
 
  ++ loss
   sorter = Loss(entries=>10)
-  title = The Loosers
+  title = The Losers
```
Comment (jsoref) [inline lib/Smokeping/sorters/Loss.pm]: Looser is a word, but it's roughly the result of loosening something.
Loser is also a word, and it's what happens when something loses.

**Comment by oetiker** [comment]
awesome

**Pull request merged**